docs/blog/newsletter-june-2019.rst

```diff
@@ -0,0 +1,142 @@
+- `MacGyvering your docs </videos/eu/2015/macgyvering-your-docs-proeland>`__ - Paul Roeland, Prague 2015
+- `Bootstrapping docs at a startup </videos/na/2017/bootstrapping-docs-at-a-startup-jesse-seldess>`__ - 
+Jesse Seldess, Portland 2017
+- `Backseat content strategy <https://www.youtube.com/watch?v=omwamA30e_Y&list=PLy70RNJ7dYrJ1wANiqa7ObwUnoJjouQjt&index=4>`__ - Laura Bailey, Australia 2018
+- `Where do I start? The art and practice of documentation triage </videos/portland/2018/where-do-i-start-the-art-and-practice-of-documentation-triage-neal-kaplan>`__ - Neil Kaplan, Portland 2018
+- `How to tear down existing documentation and rewrite docs that actually work </videos/prague/2018/how-to-tear-down-existing-documentation-and-rewrite-docs-that-actually-work-alexandra-white>`__ - Alexandra White, Prague 2018
+- `Tackling technical debt in the docs </videos/prague/2018/tackling-technical-debt-in-the-docs-louise-fahey>`__ - Louise Fahey, Prague 2018 
+- `A year in the life of the better docs project </videos/prague/2018/a-year-in-the-life-of-the-better-docs-project-rowan-cota>`__ - Rowan Cota, Prague 2018
+
+--------------------
+DITA vs docs-as-code
+--------------------
+
+Our final discussion this month comes from a community question about what the differences are between docs-as-code and DITA. The short answer is that docs-as-code is a practice, whereas DITA is a tool and a format. So in theory, you could practise docs-as-code with DITA just as you would with markdown.
+
+However, it's not necessarily that straightforward. One issue is that if you chose to use DITA, you put a higher barrier to entry on non-tech-writer contributions: reading the XML isn't tricky, and licenses for tools can be expensive. You can help others review by publishing the content to PDF, but that may not be an ideal review format.
+
+The toolchains also generally differ. Typically, DITA uses a closed content management system and toolchain, whereas docs-as-code matches what developers use for their code.
+
+DITA also emphasizes certain features, like content reuse, that don't come out of the box with most docs-as-code tools. However, you can often add those features in - Tom Johnson's `blog series comparing DITA and Jekyll <https://idratherbewriting.com/2015/03/23/new-series-jekyll-versus-dita/>`__ discusses this. He warns that with lots of customisations, you risk locking yourself into a particular toolchain - but others pointed out that it's never "easy" to change docs systems anyway.
+
+Some people pointed out the ideological differences. A lot of the point of DITA is the structure, allowing sophisticated content re-use. Docs-as-code/markdown/SSGs don’t come with the same feature set because they sometimes no have structure at all, even to the point of being anti-structure.
+
+So while you can potentially unite the two, it's worth remembering that the philosophies underlying docs-as-code and DITA are quite different.
+
```

Comment:
Beth, this is a brilliant writeup of a difficult conversation. We all knew you rocked before this, but now you rock specially in this case!

Reply:
Aw, thank you so much!